ui/lib/apps/SlowQuery/components/List.tsx

```diff
@@ -0,0 +1,209 @@
+
+  useEffect(() => {
+    async function getSlowQueryList() {
+      setLoading(true)
+      const res = await client
+        .getInstance()
+        .slowqueryListGet(
+          curSchemas.join(','),
+          desc,
+          limit,
+          curTimeRange?.end_time,
+          curTimeRange?.begin_time,
+          orderBy,
+          searchText
+        )
+      setLoading(false)
+      if (res?.data) {
+        setSlowQueryList(res.data || [])
+      }
+    }
+    getSlowQueryList()
+  }, [curTimeRange, curSchemas, orderBy, desc, searchText, limit, refreshTimes])
+
+  // TODO: refine
+  const location = useLocation()
+  useEffect(() => {
+    if (location.search === '?from=detail') {
+      // load
+      const searchOptionsStr = localStorage.getItem('slow_query_search_options')
+      if (searchOptionsStr !== null) {
+        const searchOptions = JSON.parse(searchOptionsStr)
+        console.log(searchOptions)
+        setCurTimeRange(searchOptions.curTimeRange)
+        setCurSchemas(searchOptions.curSchemas)
+        setSearchText(searchOptions.searchText)
+        setLimit(searchOptions.limit)
+        setOrderBy(searchOptions.orderBy)
+        setDesc(searchOptions.desc)
+      }
+    }
+  }, [])
+
+  function handleTimeRangeChange(val: TimeRange) {
+    setCurTimeRange(val)
+  }
+
+  function onColumnClick(_ev: React.MouseEvent<HTMLElement>, column: IColumn) {
+    if (column.key === orderBy) {
+      setDesc(!desc)
+    } else {
+      setOrderBy(column.key)
+      setDesc(true)
+    }
+  }
+
+  function handleRowClick(rec) {
+    const qs = DetailPage.buildQuery({
+      digest: rec.digest,
+      connectId: rec.connection_id,
+      time: rec.query_time,
+    })
+    navigate(`/slow_query/detail?${qs}`)
+
+    // save search options
+    const searchOptions = JSON.stringify({
+      curTimeRange,
+      curSchemas,
+      orderBy,
+      desc,
+      searchText,
+      limit,
+    })
+    console.log(searchOptions)
+    localStorage.setItem('slow_query_search_options', searchOptions)
+  }
+
+  return (
+    <ScrollablePane style={{ height: '100vh' }}>
+      <Card>
+        <div style={{ display: 'flex' }}>
+          <Space size="middle">
+            <TimeRangeSelector
+              value={curTimeRange}
+              onChange={handleTimeRangeChange}
+            />
+            <Select
+              value={curSchemas}
+              mode="multiple"
+              allowClear
+              placeholder={t('statement.pages.overview.toolbar.select_schemas')}
+              style={{ minWidth: 200 }}
+              onChange={setCurSchemas}
+            >
+              {schemas.map((item) => (
+                <Option value={item} key={item}>
+                  {item}
+                </Option>
+              ))}
+            </Select>
+            <Search
+              placeholder="包含文本"
+              value={searchText}
+              onChange={(e) => setSearchText(e.target.value)}
+            />
+            <InputNumber
```

Comment:
prefer to be a dropdown select providing some common selections like `[100, 200, 500, 1000]` because accurate number of slow log limit is not very useful